README.md

```diff
@@ -93,6 +93,27 @@ Example input and linked result:
 
 Also see [test cases](src/test/java/org/nibor/autolink/AutolinkUrlTest.java).
 
+### Simple link extraction
+
+Extract links not fully qualified URLs but still somewhat valid of the form `www.example.com`.
+
+The same heuristics applies as for the URL extraction.
+
+Examples:
+
+* `www.example.com.` → [www.example.com]().
+* `(www.example.com)` → ([www.example.com]())
+* `[..] link:www.example.com [..]` → \[..\] link:[www.example.com]() \[..\]
+
+Not supported:
+
+* Uppercase `www`'s, e.g. `WWW.example.com` and `wWw.example.com`
+* Too many or too few `w`'s, e.g. `wwww.example.com`
+
+At least one TLD is required, but multiple are allowed so `www.something.co.uk` is valid.
```

Comment:
`www.com` has a TLD (`.com`), so this could be clearer. I would just go with something like: The domain must have at least 3 parts, so `www.com` is not valid, but `www.something.co.uk` is.